pkg/api/v1beta3/types.go

```diff
@@ -379,6 +379,8 @@ type Container struct {
 	// Optional: Default to false.
 	Privileged bool `json:"privileged,omitempty"`
 	// Optional: Policy for pulling images for this container
+    // Defaults to PullAlways if the image tag is latest,
```

Comment:
Please revert. @vishh could do this with the addition of descriptions, though the descriptions are most important.